packages/config/README.md

```diff
@@ -0,0 +1,71 @@
+# Collection of Base Configurations for Monorepo
+
+The package name is `@cloudoperators/juno-config`.
+
+- Typescript
+- ESlint
+
+## Typescript
+
+This package provides a base TypeScript configuration to be used across multiple packages in a monorepo setup. It ensures consistent TypeScript settings and simplifies the TypeScript setup process for each package.
+
+## Usage
+
+You can use this configuration as a starting point in any package or app within your monorepo using TurboRepo.
```

Comment:
Does this sentence mean that other people could use this setup in their own monorepos? Or did you mean "package or app with **this** monorepo?

Reply:
package or app within our monorepo